.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ jobs:
           working_directory: packages/contracts-bedrock
       - run:
           name: upload coverage
-          command: codecov --verbose --clean --flag contracts-bedrock-forge
+          command: codecov --verbose --clean --flag contracts-bedrock-tests
```

Comment:
all the names here should also appear in [codecov.yml](https://github.com/ethereum-optimism/optimism/blob/8086bd2cf96d86fb58aa38c6677bd21f060b3d9e/codecov.yml#L1)